docs/CHANGELOG.html

```diff
@@ -58,7 +58,7 @@ <h1 data-sassdoc="project-title">
 
   
     
-  <nav role="navigation" data-nav-menu="small" data-toggle="target" id="nav-small" data-target-id="nav-small" aria-expanded="false" data-auto-closing="false" data-auto-closing-on-any-click="false">
+  <nav data-nav-menu="small" data-toggle="target" id="nav-small" data-target-id="nav-small" aria-expanded="false" data-auto-closing="false" data-auto-closing-on-any-click="false">
 
     <div data-sassdoc="nav-subsection">
```

Comment:
This is not a big deal, but it's not easy to tell the two subtitle sizes apart from one another. Maybe make the smaller one not bold?